addons/extension_main_menu/functions/fnc_onLoad.sqf

```diff
@@ -0,0 +1,42 @@
+#define DEBUG_SYNCHRONOUS
```

Comment:
Is this debug needed in production? The same applies to other files.

Reply:
It's not a debug, this changes how CBA logging works: https://github.com/CBATeam/CBA_A3/blob/9191ad571f4e31b6e7c8e17a407f0a2b2296d2c8/addons/main/script_macros_common.hpp#L173

By default the CBA logging is done via scheduled loop, I guess this is a remnant from the times when logging was done in game thread and could cause fps drops when done a lot in unscheduled. (https://github.com/CBATeam/CBA_A3/blob/1f780aee7f291e7598e0855bc8ba38b1df5b0d67/addons/diagnostic/fnc_log.sqf)